src/Components/AdditionalRepositoriesTable/AdditionalRepositoriesTable.js

```diff
@@ -71,8 +79,8 @@ const AdditionalRepositoriesTable = (props) => {
   };
 
   const countProducts = () => {
-    const filtedRepo = sortedRepositories;
-    return filtedRepo?.length;
+    const filteredRepo = sortedRepositories;
+    return filteredRepo?.length;
   };
```

Comment:
I know this function wasn't introduced in this story - but since something was changed on it I'll bring it up. I don't see the point of it - couldn't we just check `sortedRepositories?.length` anywhere we need to?